lib/web/desktop.go

```diff
@@ -392,6 +392,33 @@ func proxyWebsocketConn(ws *websocket.Conn, wds net.Conn) error {
 	return trace.NewAggregate(retErrs...)
 }
 
+// wasWebsocketClosedByClient handles the error returned by proxyWebsocketConn
+func handleProxyWebsocketConnErr(err error, log *logrus.Entry) {
+	if err == nil {
+		return
+	}
+
+	if agg, ok := trace.Unwrap(err).(trace.Aggregate); ok {
```

Comment:
Handle `!ok` in the if and return immediately, so we remove the nesting?

https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
d9d1d6f87852a508b34da1da7665c013ff785feb